### PR TITLE
feat/testing: complete

### PR DESCRIPTION
**Pull request by SabiMantock**
main features: 
- Set up the project with `create-react-app` and linters for JavaScript and CSS
- Set up reducer toolkit and configured redux store
- Wrote action creators for dota heroes
- Fetched hero data from openDota API
- Rendered a list of heroes on the Home Page
- Render hero details on the HeroDetail Page
- Wrote unit tests for pure functions inside redux and integration tests using React Testing Library.

### CODE REVIEW

File: src/tests/App.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          <BrowserRouter>
+            <App />
+          </BrowserRouter>
+        </Provider>
+      </React.StrictMode>,
+    );
+
+    root.unmount();
+  });
+});
```
Comment (levy002) [inline src/tests/App.test.js:27]: - [ ] KIndly make sure to add more unit tests for your components and pure functions in the Redux code.
- [ ] Kindly make sure to add integration tests using React Testing Library.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @SabiMantock   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- GOOD UI :heavy_check_mark: 
- Well-structured codebase :heavy_check_mark: 
- Professional README :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) in your `Acknowledgments` section.

- [ ] Kindly make sure to update the PR description and make sure to describe every single detail of what you did on that project as long this is the main PR of the finished project.

   - ### GOOD PR DESCRIPTION
   
    image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -166,7 +179,7 @@ npm run build
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-- Add other Player Stats
+- Add Player Stats
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (1GORDON) [inline README.md:185]: Great job @SabiMantock 🤝

- [ ] [Optional] Kindly fix the broken issues link to make your readme file more professional.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @SabiMantock ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**